src/main/java/net/snowflake/ingest/streaming/internal/SnowflakeStreamingIngestClientInternal.java

```diff
@@ -586,4 +618,35 @@ FlushService getFlushService() {
   ParameterProvider getParameterProvider() {
     return parameterProvider;
   }
+
+  /**
+   * This method is called to fetch an object name for all registered metrics. It can be called
+   * during registration or unregistration. (Internal implementation of codehale)
+   *
+   * @param clientName name of the client. Passed in builder
+   * @param jmxDomain JMX Domain
+   * @param metricName metric name used while registering the metric.
+   * @return Object Name constructed from above three args
+   */
+  static ObjectName getObjectName(String clientName, String jmxDomain, String metricName) {
+    try {
+      StringBuilder sb = new StringBuilder(jmxDomain).append(":clientName=").append(clientName);
+
+      sb.append(",name=").append(metricName);
+
+      return new ObjectName(sb.toString());
+    } catch (MalformedObjectNameException e) {
+      logger.logWarn("Could not create Object name for MetricName:{}", metricName);
```

Comment:
We try to match the server side coding standard, so please use =
```suggestion
      logger.logWarn("Could not create Object name for MetricName={}", metricName);
```

Reply:
sure will do!